src/substrait/textplan/converter/tests/BinaryToTextPlanConversionTest.cpp

```diff
@@ -120,6 +119,73 @@ std::vector<TestCase> GetTestCases() {
                        "  function multiply:opt_fp64_fp64 as multiply;\n"
                        "}\n"),
       },
+      {
+          "read local files",
+          R"(relations: {
+            root: {
+              input: {
+                read: {
+                  local_files {
+                    items {
+                      partition_index: 0
+                      start: 0
+                      length: 3719
+                      uri_file: "/mock_lineitem.orc"
+                      orc {}
+                    }
+                  }
+                }
+              }
+            }
+          })",
+          AllOf(
+              HasSymbols({"local", "read", "root"}),
+              SerializesTo(
+                  "read relation read {\n"
+                  "}\n"
+                  "\n"
+                  "source local_files local {\n"
+                  "  items = [\n"
+                  "    {uri_file: \"/mock_lineitem.orc\" length: 3719 orc: {}}\n"
```

Comment:
I've pulled in the changes from the next PR to use raw strings in the test input here.  However the test output isn't using them because I would like to use a whitespace ignoring compare for the output so it won't matter what the formatting is here in the tests.  SerializeContains was partially addressing this issue so I will investigate ignoring whitespace when I revisit that implementation later on.